### PR TITLE
add external logger

### PR DESCRIPTION
**Pull request by joeydotdev**

### CODE REVIEW

**Review by tanndlin — APPROVED** [review]
I trust you lol

File: scripts/logger.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+function join() {
+  ensureLogger();
+  return through((data) => {
+    logger.info(data);
+  });
+}
+
+process.stdin.pipe(split()).pipe(join()).pipe(process.stdout);
```
Comment (c650) [inline scripts/logger.js:33]: What is split

**Review by c650 — COMMENTED** [review]
Please also capture stderr 

otherwise Makes sense to me.

id like a comment on what the split function is 

also we might as well strip down the logger service

File: Dockerfile [hunk 2]
```diff
@@ -5,4 +5,4 @@ COPY package*.json ./
 RUN npm install
 COPY . .
 RUN npm run build
-CMD ["node", "./dist/index.js"]
+CMD "node ./dist/index.js | node ./scripts/logger.js"
```
Comment (c650) [inline Dockerfile:8]: Put a 2 >& 1 before the pipe

**Review by c650 — APPROVED** [review]
LGTM as long as you ran it in docker and it works

**Comment by tanndlin** [comment]
can u change the branch so its on develop

**Pull request merged**